<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EndSession</class>
 <widget class="QWidget" name="EndSession">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1237</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="tTitleLabel" name="titleLabel">
     <property name="text">
      <string>End Session</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="centeredWidget" native="true">
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QLabel" name="descriptionLabel">
            <property name="text">
             <string>What would you like to do now?</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="powerOffButton">
          <property name="text">
           <string>Power Off</string>
          </property>
          <property name="icon">
           <iconset theme="system-shutdown">
            <normaloff>../../thedesk/desktop/session</normaloff>../../thedesk/desktop/session</iconset>
          </property>
          <property name="description">
           <string>Turns off your computer.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="rebootButton">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="text">
           <string>Reboot</string>
          </property>
          <property name="icon">
           <iconset theme="system-reboot">
            <normaloff>../../thedesk/desktop/session</normaloff>../../thedesk/desktop/session</iconset>
          </property>
          <property name="description">
           <string>Turns off your computer and turns it on again</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="suspendButton">
          <property name="text">
           <string>Suspend</string>
          </property>
          <property name="icon">
           <iconset theme="system-suspend">
            <normaloff>../../thedesk/desktop/session</normaloff>../../thedesk/desktop/session</iconset>
          </property>
          <property name="description">
           <string>Puts your computer in a low power state so it reopens quickly</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exitButton">
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tTitleLabel</class>
   <extends>QLabel</extends>
   <header location="global">ttitlelabel.h</header>
   <slots>
    <signal>backButtonClicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
